weblate/trans/templatetags/translations.py

```diff
@@ -603,9 +603,16 @@ def unit_state_title(unit) -> str:
 def try_linkify_filename(
     text, filename: str, line: str, unit, profile, link_class: str = ""
 ):
-    """Attemp to convert `text` to a repo link to `filename:line`."""
+    """
+    Attempt to convert `text` to a repo link to `filename:line`.
+
+    If the `text` is prefixed with a protocol, like https://, http://, ftp://, the
+    link will be an absolute link to the specified resource.
+    """
     link = None
-    if profile:
+    if re.search(r"\w+://", text):
```

Comment:
I don't think we want to whitelist all protocols here. I'd stick with `http://` and `https://` only.

Reply:
Agreed and changed!